content/doc/book/security/securing-org-folders-and-multibranch-pipelines.adoc

```diff
@@ -0,0 +1,81 @@
+
+== Affected scenarios
+
+The above concerns may not matter to users with simple security models.
+
+You should consider the above implications carefully if:
+
+* You use a single organization folder in Jenkins to build multiple SCM repositories for which the users with SCM access to one repository must not have access to the other repositories being built.
+* Your SCM users must only have read/write access to repositories and no additional permissions.
+
+Note that if you allow untrusted users to create or configure jobs in locations where credentials are available, this issue does not matter, as those users can use the credentials directly.
+
+Additionally, these concerns only apply to users who do not have Job/Configure permission (or more powerful permissions) in Jenkins, but who do have the ability to edit Pipelines, for example because they have write access to repositories that contain Jenkinsfiles.
+If untrusted users do not have the ability to edit Jenkinsfiles, for example because Multibranch Pipelines are configured to use a centralized Pipeline via a plugin such as https://plugins.jenkins.io/inline-pipeline/[inline-pipeline] or https://plugins.jenkins.io/pipeline-multibranch-defaults/[pipeline-multibranch-defaults], these issues do not matter.
+
+== Solutions and workarounds
+
+=== Generic workarounds
+
+If your primary concern is to prevent users with SCM read/write access to a repository from using the scan credentials to access other repositories being built by an Organization Folder, you have a few options:
+
+* Assuming that your SCM provider allows you to create credentials which may only access specific repositories (such as GitHub's fine-grained personal access tokens), you can create multiple Organization Folders configured to build distinct subsets of the repositories in question, defining the restricted credentials directly on each Organization Folder in Jenkins.
+* Depending on your SCM provider's credential-related features, you may not be able to use Organization Folders securely, and would instead need to create one Multibranch Pipeline for each repository with a credential defined on it that only allows access to that specific repository.
+
+Unfortunately, if your primary concern is to prevent users with SCM read/write access to a repository from using the scan credentials to gain elevated permissions, there is no generic solution.
+In some cases, you may be able to remove permissions from the credential to make it less powerful.
+For example, you may be able to remove webhook-related permissions from the credentials, which will prevent Jenkins from automatically managing webhooks, in favor of having an administrator configure webhooks manually.
+
+=== Enhanced GitHub App Credentials
+
+If you are using GitHub Branch Source Plugin with GitHub App Credentials, the plugin offers various features as of version TODO that can be used to improve security.
+See TODO.
```

Comment:
I guess the best way to deal with the dependency in documentation links between the PRs would be to temporarily comment this out, merge this so that the `github-branch-source` PR can reference this page, then come back and uncomment this and link to the updated `github-branch-source` docs.